### ISSUE TITLE
hello.htmlにformをつくってみましょう。

### ISSUE THREAD

**Issue by yoshi-koyama** [HtinSharKyaw/SpringHelloWorld#7]
下のが下の画像のようなformを作ってみてください。

従業員＝じゅうぎょういん
入力画面＝にゅうりょくがめん
年齢＝ねんれい
部署＝ぶしょ
性別＝せいべつ
入社年月日＝にゅうしゃねんがっぴ
退社年月日＝たいしゃねんがっぴ
送信＝そうしん

<img width="963" alt="スクリーンショット 2020-03-22 14 37 10" src="https://user-images.githubusercontent.com/62045457/77243181-a03db600-6c4a-11ea-9be8-0b6d9c96e2d9.png">

**Comment by yoshi-koyama** [comment]
次は必ずpull requestをあげましょう！

`git checkout -b topic/employee-form`

create employee-form.html and update HomeController.java

`git add .`
`git commit -m "please think about the commit message"`
`git push --set-upstream origin topic/employee-form`

and make pull request

**Comment by HtinSharKyaw** [comment]
はい。わかりました。ありがとうございます。よろしくお願いいたします。 Sent from Mail for Windows 10 From: yoshi-koyamaSent: Sunday, March 22, 2020 12:12 PMTo: HtinSharKyaw/SpringHelloWorldCc: SubscribedSubject: Re: [HtinSharKyaw/SpringHelloWorld] hello.htmlにformをつくってみましょう。 (#7) 次は必ずpull requestをあげましょう！git checkout -b topic/employee-formcreate employee-form.html and update HomeController.javagit add .git commit -m "please think about the commit message"git push --set-upstream origin topic/employee-formand make pull request—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by yoshi-koyama** [comment]
わかりました。

だけで大丈夫ですよ！
